dolphinscheduler-ui-next/src/views/resource/task-group/option/use-table.ts

```diff
@@ -139,14 +138,8 @@ export function useTable(
             }
 
             item.projectName = projectName
-            item.createTime = format(
-              parseTime(item.createTime),
-              'yyyy-MM-dd HH:mm:ss'
-            )
-            item.updateTime = format(
-              parseTime(item.updateTime),
-              'yyyy-MM-dd HH:mm:ss'
-            )
+            item.createTime = simpleDateFormat(item.createTime)
```

Comment:
`createTime` and `updateTime` look like not be format. it will be format in table render.